src/sdk/bucketspace_fs.js

```diff
@@ -164,17 +165,58 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
         }
     }
 
+    /**
+     * check_bucket_config will change the bucket_storage_path
+     * and - TODO: Bucket owner check
+     * @param {object} bucket
+     * @returns {Promise<boolean>}
+     */
```

Comment:
I meant that there is no reason to call this function check_bucket_config if it's checking the bucket storage path, there is no relation to the config here

Reply:
I changed it, I moved the TODO that was there that it won't be missed (we can open an issue with all the details if you want).